.github/workflows/tests.yml

```
@@ -0,0 +1,24 @@
+name: tests
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "master" ]
```
we can minimise the number of triggers